en/core-api/bank-transfer.md

```diff
@@ -1,69 +1,53 @@
 # Core API Bank Transfer Integration
```

Comment:
The formatting and structure of this page is broken, please learn from the other feedbacks and fix this page.